<?xml version="1.0"?>
<launch>

  <node pkg="aruco_fractal_ros" name="aruco_fractal_ros_node" type="aruco_fractal_ros_node" output="screen">
	<param name="image_topic_name" value="/csi/image_raw" />
	<param name="camera_info_topic_name" value="/csi/camera_info" />
	<param name="marker_topic_name" value="/aruco/aruco" />
    <param name="rate" value="30" />

	<param name="window_size" value="10" />
	<param name="marker_size" value="0.30" />
	<param name="parent_frame" value="map" />
	<param name="marker_frame" value="marker" />
	<param name="marker_config" value="FRACTAL_4L_6" />
	<param name="display_image" value="true" />
	<param name="invert_image" value="true" />

<!-- 	<param name="calibration_path" value="$(find aruco_fractal_ros)/config/d435_720p.yaml"/>  -->
 </node>

</launch>


